src/abbfreeathome/devices/switch_sensor.py

```diff
@@ -18,6 +18,18 @@ class DimmingSensorLongpressState(enum.Enum):
     longpress_down_release = "0"
 
 
+class DimmingSensorCombinedState(enum.Enum):
+    """An Enum class for the combined state."""
+
+    unknown = None
+    longpress_up_press = "9"
+    longpress_up_release = "8"
+    longpress_down_press = "1"
+    longpress_down_release = "0"
+    shortpress_up = 2
+    shortpress_down = 3
```

Comment:
Since this is used for both switching and dimming the name probably needs to reflect that. You've also got a mix of string and int, and at this point since it's mixing two different pairing id's I think the values can be arbitrary.

```suggestion
class SwitchingDimmingSensorState(enum.Enum):
    """An Enum class for the switch and dimming state."""

    unknown = None
    shortpress_up = 0
    shortpress_down = 1
    longpress_up = 2
    longpress_up_release = 3
    longpress_down = 4
    longpress_down_release = 5
```